cpp/src/arrow/flight/sql/server_test.cc

```diff
@@ -381,6 +382,43 @@ TEST_F(TestFlightSqlServer, TestCommandGetTablesWithIncludedSchemas) {
   AssertTablesEqual(*expected_table, *table);
 }
 
+TEST_F(TestFlightSqlServer, TestCommandGetTypeInfo) {
+  ASSERT_OK_AND_ASSIGN(auto flight_info, sql_client->GetTypeInfo({}));
+
+  ASSERT_OK_AND_ASSIGN(auto stream,
+                       sql_client->DoGet({}, flight_info->endpoints()[0].ticket));
+
+  auto expected_schema = SqlSchema::GetTypeInfoSchema();
+
+  const std::shared_ptr<RecordBatch>& batch =
```

Comment:
nit, but this isn't a reference either - it should just be `const std::shared_ptr<RecordBatch> batch` or `const auto batch`

Reply:
(And the same goes below)

Reply:
Fixed